conda/dev.yml

```diff
@@ -5,30 +5,30 @@ channels:
 dependencies:
   # Base
   # =================
+  - python=3.9.13
+  - pip=22.2.2
```

Comment:
I moved `python` and `pip ` to the top because they are usually at the top since they are  "system" level dependencies required to install the other packages.